rush.json

```diff
@@ -125,7 +119,6 @@
    * LTS versions: https://nodejs.org/en/download/releases/
    */
   "nodeSupportedVersionRange": ">=16.13.0 <17.0.0 || >=18.12.0 <19.0.0",
-
```

Comment:
Please don't include unnecessary formatting changes in PRs

Reply:
This is automatically done by rush format as a pre-commit hook due to me changing this file.